plugins/janus_videoroom.c

```diff
@@ -376,16 +378,29 @@ room-<unique room ID>: {
 			"room" : <unique numeric ID>,
 			"description" : "<Name of the room>",
 			"pin_required" : <true|false, whether a PIN is required to join this room>,
+			"is_private" : <whether this room is 'private' (as in hidden) or not>,
```

Comment:
This is missing `true|false, ` to be consistent with others (like the one above).

Reply:
done